images/tier2/change-file.svg

```diff
@@ -1,5 +1,5 @@
 <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" viewBox="0 0 24 24">
```

Comment:
Did you intend to modify these 3 SVGs at the same time? Seems like a separate thing maybe for a separate PR?